modules/features/filters/src/main/java/au/com/shiftyjelly/pocketcasts/filters/CreateFilterViewModel.kt

```diff
@@ -24,9 +26,24 @@ import javax.inject.Inject
 import kotlin.coroutines.CoroutineContext
 
 @HiltViewModel
-class CreateFilterViewModel @Inject constructor(val playlistManager: PlaylistManager, val episodeManager: EpisodeManager, val playbackManager: PlaybackManager) :
-    ViewModel(),
+class CreateFilterViewModel @Inject constructor(
+    val playlistManager: PlaylistManager,
+    val episodeManager: EpisodeManager,
+    val playbackManager: PlaybackManager,
+    private val analyticsTracker: AnalyticsTrackerWrapper
+) : ViewModel(),
     CoroutineScope {
+
+    companion object {
+        private object AnalyticsPropKey {
+            const val DID_CHANGE_AUTO_DOWNLOAD_EPISODE_COUNT = "did_change_episode_count"
+            const val DID_CHANGE_ICON = "did_change_icon"
+            const val DID_CHANGE_NAME = "did_change_name"
+            const val DID_CHANGE_AUTO_DOWNLOAD = "did_change_auto_download"
+            const val DID_CHANGE_COLOR = "did_change_color"
+        }
```

Comment:
I kind of liked how this grouped the related strings together and kind of gave them their own namespace. I thought this could be really nice in classes that had other unrelated strings in the companion object. wdyt?

Reply:
I was thinking about it lately and I like this idea. 👍 
I'll start grouping properties like this in future PRs.